src/vsr/replica_test.zig

```diff
@@ -471,9 +471,10 @@ test "Cluster: repair: partition 2-1, then backup fast-forward 1 checkpoint" {
     try expectEqual(r_lag.status(), .normal);
     try expectEqual(r_lag.op_checkpoint(), 0);
 
-    // Allow repair, but ensure that state sync doesn't run.
-    r_lag.drop(.__, .bidirectional, .sync_checkpoint);
```

Comment:
Caught the bug when removing `.sync_checkponit` for https://github.com/tigerbeetle/tigerbeetle/pull/2254!